src/JustEat.StatsD/IpTransport.cs

```diff
@@ -15,21 +15,14 @@ public IpTransport(IPEndPointSource endPointSource)
         }
 
         public void Send(string metric)
-        {
-            Send(new[] {metric});
-        }
-
-        public void Send(IEnumerable<string> metrics)
         {
             var endpoint = _endpointSource.GetEndpoint();
-            var packets = metrics.ToMaximumBytePackets();
+            var bytes = Encoding.UTF8.GetBytes(metric);
 
             using (var socket = MakeIpDatagramSocket())
             {
-                foreach (var packet in packets)
-                {
-                    socket.SendTo(packet, endpoint);
```

Comment:
Even if we only sent one metric, this code would have split the string into 512-byte packets if required. Is that something we don't need to worry about anymore?

Reply:
As I understand it, you can't split one metric over multiple packets. The packets are independent of each other and there's no state shared between them. Each packet should contain one or more valid metrics.

In `PacketBuilder` line 43-ish, I think it means that if the metric text is larger than the desired size for a packet, then you get an extra-large packet and there's no way around that in this code.

Reply:
OK cool. It just looked like that was how it worked at a high-level.